src/msg/async/Stack.h

```diff
@@ -245,6 +246,7 @@ class Worker {
     plb.add_u64_counter(l_msgr_send_bytes, "msgr_send_bytes", "Network sent bytes", NULL, 0, unit_t(UNIT_BYTES));
     plb.add_u64_counter(l_msgr_active_connections, "msgr_active_connections", "Active connection number");
     plb.add_u64_counter(l_msgr_created_connections, "msgr_created_connections", "Created connection number");
+    plb.add_u64_counter(l_msgr_faulted_connections, "msgr_faulted_connections", "Number of faulted connections", NULL, PerfCountersBuilder::PRIO_USEFUL, unit_t(UNIT_BYTES));
```

Comment:
Why `unit_t(UNIT_BYTES)`? The intuition suggests the new counter should follow e.g. `msgr_created_connections`.

Reply:
tbh I just copied and pasted, I'll remove it